mirrord/layer/src/file.rs

```diff
@@ -218,6 +232,11 @@ impl FileHandler {
                 debug!("DaemonMessage::XstatResponse {:#?}!", xstat);
                 pop_send(&mut self.xstat_queue, xstat)
             }
+            ReadDir(read_dir) => {
+                debug!("DaemonMessage::ReadDirResponse {:#?}!", read_dir);
```

Comment:
Is this intentionally still in `debug` or should this be changed to `trace`:
```suggestion
                trace!("DaemonMessage::ReadDirResponse {:#?}!", read_dir);
```
personally I have nothing against leaving interesting logs as `debug`, as long as they are not spammy and it's intentional.